README.md

```diff
@@ -80,22 +92,17 @@ npm install
 npm run testrpc
 ```
 
-#### Terminal 3: Start the testing Python stub
+#### Terminal 3: Python client
 
 ```bash
 # Configure your environment
 . ./setup_env.sh
 
 cd pyClient
-# Follow the few steps described in the README of the python stub
 ```
 
-### Use the pyClient
-
-This Proof of Concept comes with some minimal building blocks to integrate Zeth with your applications.
-You can use the [python client](pyClient/README.md) to interact with the proving service and request proofs on a given `(instance, witness)` pair.
-
-## Secure Multi Party Computation for the Groth16 SRS generation
```

Comment:
Please add this title back otherwise the mpc section gets added to the section about the python client

Reply:
Oops.  Must have been accidentally removed somewhere.  Replaced.